README.rst

```diff
@@ -1872,6 +1872,20 @@ provided it uses Apipie as a backend.
 
 And if you write one on your own, don't hesitate to share it with us!
 
+====================
+ Contributing
+====================
+
+Since this gem does not have a Gemfile, you need to specify it in your shell with:
+
+.. code:: shell
+   BUNDLE_GEMFILE='gemfiles/Gemfile.rails61'
```

Comment:
Thank you for this. you're right, this is helpful for new people.
Maybe we should keep a default Gemfile, I recently removed it and honestly, I don't remember why it felt redundant.

but this is the correct way to specify the rails version you'd expect.

that said, these new code blocks don't show up when I try to view the file in rendered mode.
maybe we need to specify `bash` instead of `shell` ?

I'm not familiar with `.rst` format actually.

Reply:
it was a dead symlink after I moved the gemfiles 39af442f2995e4555fb568216c6d790d923ff06b

so I deleted it